### ISSUE TITLE
Conda install ignite 

### ISSUE THREAD

**Issue by RoyHirsch** [pytorch/ignite#466]
Hi,
I'm trying to download view pip, I've downloaded the current version of 0.1.2 but don't have the current master. For example I don't have the is_multiclass flag. How can I make sure I download the updated version in the repo ? 
I've used : pip install pytorch-ignite

Thanks,
Roy

**Comment by vfdev-5** [comment]
Hi Roy, you can install the master using the following command:
```
pip install --upgrade git+https://github.com/pytorch/ignite
```

If you are using pytorch-nightly, we have ignite nightly builds with
pytorch-nightly as dependency:
```
pip install pytorch-ignite-nightly
```

HTH

**Comment by RoyHirsch** [comment]
Thank you. I would like to use the Recall/Persicion measurement for a multilabel task. But I would like to compute thre recall per class. while using th is_multilabel flag I have two options:
- is avg is True: average recall for all the samples
- is avg is Fasle: reports recall per sample.

Is there a support in reporting recall per class ?
Thank you,
Roy

**Comment by vfdev-5** [comment]
@RoyHirsch could you please detail which task would you like to do:
- multiclass = target is [0, 1, 2, ..., C]
- multilabel = target are "tags", e.g. [[0, 1, 0], [0, 1, 1], ...]

In case of multiclass, the argument `avg=False` gives Precision/Recall per class, otherwise they are averaged over all classes.

In case of multilabel, AFAIK and remember we can not compute metrics per class but per sample. Averaging is thus going per samples. Take a look for example, https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html#sklearn.metrics.recall_score with `average="samples"`.

**Comment by RoyHirsch** [comment]
My task is multilabel, means the ground true is binary vector with arbitrary True values: [[0, 1, 0], [0, 1, 1], ...]

My prediction is a scores vector in shape NUM_CLASSES, I am processing the vector into topK predictions per sample (we can assume it is processeed into bianry vector, where ony the topK columns are True and the rest is False).
I would like to report the per class recall following:

![image](https://user-images.githubusercontent.com/26789321/55289334-1445d500-53ce-11e9-9c41-045c96128e75.png)
*DeepConvolutionalRankingfor MultilabelImageAnnotation

Maybe I am missing something out , but i think this is a relevant metric.
Roy

**Comment by vfdev-5** [comment]
I see, thanks for the reference. I have an impression that they just measure each tag independently. I'm not sure if we have a metric that could be hacked to produce top-k per-class precision/recall but yes I agree this can be a relevant metric and definitly is needed to be added to Ignite.

If you would like to implement it and send a PR, would be nice ! Community contributions are very welcome :)

**Comment by RoyHirsch** [comment]
Thank youy for the quick help ! I will ipdate if I will have the resources to contribute a generalized code.
Roy

**Comment by vfdev-5** [comment]
@RoyHirsch  I close this issue as installation problem is solved. We can use #467 to discuss about the proposition of the new metric.
